unlock-app/src/components/interface/buttons/lock/Copy.js

```diff
@@ -0,0 +1,11 @@
+import Svg from '../../svg'
+import React from 'react'
+import { LockButton } from '../Button'
+
+const Copy = () => (
```

Comment:
This does not actually have the copy logic, does it?

Reply:
Also maybe worth passing the props like you did on the other PR from yesterday?

Reply:
Good call about the props - but the copy logic is still to come in a subsequent PR once this one clears. I have it ready to go but it builds on this one.

Reply:
Works for me!